client/examples/tutorial.rs

```diff
@@ -0,0 +1,186 @@
+//! This file contains examples from the Rust tutorial.
+//! <https://hyperledger.github.io/iroha-2-docs/guide/rust.html#_2-configuring-iroha-2>
+//!
+#![allow(clippy::restriction, clippy::needless_borrow)]
+
+// use eyre::{Error, WrapErr};
+// BEGIN FRAGMENT: rust_config_example
+// use iroha_config::client::Configuration;
+
+fn main() {
+    // let config = Configuration::from_path("configs/client_cli/config.json")
+    //     .wrap_err("Failed to parse `../configs/client_cli/config.json`")
+    //     .expect("Verified in tests");
+    // // Your code goes here…
+
+    // // BEGIN ESCAPE
+    // json_config_client_test(&config)
+    //     .expect("JSON config client example is expected to work correctly");
+    // domain_registration_test(&config)
+    //     .expect("Domain registration example is expected to work correctly");
+    // account_definition_test().expect("Account definition example is expected to work correctly");
+    // account_registration_test(&config)
+    //     .expect("Account registration example is expected to work correctly");
+    // asset_registration_test(&config)
+    //     .expect("Asset registration example is expected to work correctly");
+    println!("Success!");
+    // END ESCAPE
+}
+// END FRAGMENT
+
+// fn json_config_client_test(config: &Configuration) -> Result<(), Error> {
```

Comment:
The entire point of these tests is that we need to not have them commented. Uncomment please. If they fail we need to fix them.